javnov.Selenium.Axe/javnov.Selenium.Axe/AxeBuilder.cs

```diff
@@ -114,14 +129,8 @@ public JObject Analyze()
                 command = $"axe.a11yCheck(document, {Options}, arguments[arguments.length - 1]);";
             }
 
-            return Execute(command);
-        }
-
-        private JObject Execute(string command, params object[] args)
-        {
-            _webDriver.Manage().Timeouts().SetScriptTimeout(TimeSpan.FromSeconds(30));
-            object response = ((IJavaScriptExecutor)_webDriver).ExecuteAsyncScript(command, args);
-            return new JObject(response);
+            var violations = Execute(command).SelectToken("violations");
```

Comment:
Split in two lines, like this:
```csharp
JObject executionResult = Execute(command);
var violations = executionResult.SelectTokect("violations")
```

Also **stop using var for unknown types**, in this case we don't know which **violations** variable type is just reading the code.